### PR TITLE
Unify Wording in 3.2.0 DevOps ZH Docs

### PR DESCRIPTION
**Pull request by Felixnoo**
Signed-off-by: Felixnoo <felixliu@kubesphere.io>

/cc @kubesphere/sig-devops 
Since the UI text on KubeSphere 3.2.0 web console is updated and `DevOps project` is translated into `DevOps 项目` in Chinese, I updated relevant ZH documents accordingly in this PR.
Please take a look, thanks.

### CODE REVIEW

File: content/zh/docs/devops-user-guide/examples/a-maven-project.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ weight: 11430
 - 您需要有一个 [Docker Hub](http://www.dockerhub.com/) 帐户。
 - 您需要创建一个企业空间、一个 DevOps 项目和一个用户，并需要邀请该用户至 DevOps 项目中并赋予 `operator` 角色。有关更多信息，请参见[创建企业空间、项目、用户和角色](../../../quick-start/create-workspace-and-project/)。
```
Comment (mangoGoForward) [inline content/zh/docs/devops-user-guide/examples/a-maven-project.md:13]: The link of `create-workspace-and-project` seems linked to a '404' error page, it should be '../../../quick-start/create-workspace-and-project.md'

File: content/zh/docs/devops-user-guide/examples/a-maven-project.md [hunk 2]
```diff
@@ -73,7 +73,7 @@ kubectl get cm -n kubesphere-devops-system ks-devops-agent -o yaml
 
 在本示例中，所有工作负载都部署在 `kubesphere-sample-dev` 项目中。您必须事先创建 `kubesphere-sample-dev` 项目。
 
-### 为 Maven 工程创建一个流水线
+### 为 Maven 项目创建一个流水线
 
 1. 在您的 DevOps 项目中，转到**流水线**页面并点击**创建**，创建一个名为 `maven` 的流水线。有关更多信息，请参见[使用图形编辑面板创建流水线](../../how-to-use/create-a-pipeline-using-graphical-editing-panel)。
```
Comment (mangoGoForward) [inline content/zh/docs/devops-user-guide/examples/a-maven-project.md:78]: The link of `create-a-pipeline-using-graphical-editing-panel` should be `../../devops-user-guide/how-to-use/create-a-pipeline-using-graphical-editing-panel.md`

**Comment by mangoGoForward** [comment]
Seems no problem. Maybe we can create a new issue to solve those link error.

**Comment by JohnNiang** [comment]
Hi @mangoGoForward , thanks for your patient review. 

I suggest clicking the `Details` link to preview the document to check if the link is correct:

image

File: content/zh/docs/devops-user-guide/examples/a-maven-project.md (hunk 1 above)
Comment (Felixnoo) [reply content/zh/docs/devops-user-guide/examples/a-maven-project.md:13]: Hi, thanks for your review. The link here is correct.

File: content/zh/docs/devops-user-guide/examples/a-maven-project.md (hunk 2 above)
Comment (Felixnoo) [reply content/zh/docs/devops-user-guide/examples/a-maven-project.md:78]: Hi, thanks for your review. The link here is correct.

**Comment by mangoGoForward** [comment]
> Hi @mangoGoForward , thanks for your patient review.
> 
> I suggest clicking the `Details` link to preview the document to check if the link is correct:
> 
> image

Thanks, it helped me a lot, and looks no problem. I clicked the link in markdown file, and redirect to an error page, so I leave those comments, Do we need to process this scene?

File: content/zh/docs/devops-user-guide/examples/a-maven-project.md (hunk 1 above)
Comment (mangoGoForward) [reply content/zh/docs/devops-user-guide/examples/a-maven-project.md:13]: Sorry for the trouble.

**Comment by JohnNiang** [comment]
> Do we need to process this scene?

I think it would be a huge change. At present, I suggest not to handle this issue.

File: content/zh/docs/devops-user-guide/examples/a-maven-project.md (hunk 1 above)
Comment (Felixnoo) [reply content/zh/docs/devops-user-guide/examples/a-maven-project.md:13]: No worries.

**Pull request merged**